### PR TITLE
Add occ app:list and ldap:show-config to issue_template.md

### PR DESCRIPTION
**Pull request by ghost**
Available since OC6 so should be safe to add it.

### CODE REVIEW

**Comment by RealRancor** [comment]
Just the issue_template.md but anyway: MIT licensed

**Comment by MorrisJobke** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
- Should be ownCloud
- the command is different on various distros - let's add a e.g.

**Comment by RealRancor** [comment]
@DeepDiver1975 Sure, will change that.

Any opinion about the "LDAP configuration" part. This probably can be replaced by:

occ ldap:show-config

or is this command missing some infos which are part of the database query?

**Comment by DeepDiver1975** [comment]
I honestly have no clue about ldap @blizzz

**Comment by MorrisJobke** [comment]
@LukasReschke How is your report app going?

**Comment by blizzz** [comment]
@RealRancor yes, the occ command for LDAP please :)

**Comment by RealRancor** [comment]
@blizzz The occ command is only available since OC7 so better leave the sql queries in there?

**Comment by MorrisJobke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
@owncloud-bot retest this please

File: issue_template.md [hunk 1]
```diff
@@ -40,9 +46,17 @@ Insert your config.php content here
...
+On ownCloud 7+ with access to your command line run e.g.:
+sudo -u www-data php occ ldap:show-config
+from within your ownCloud installation folder
+
+On ownCloud 6 with access to your command line run e.g.:
+sqlite3 data/owncloud.db or mysql -u root -p owncloud
 then execute:  select * from oc_appconfig where appid='user_ldap';
 
+Without access to your command line download the data/owncloud.db to your local
+computer or access your SQL server remote and run the select query above.
```
Comment (Xenopathic) [inline issue_template.md]: remote*ly*

**Comment by Xenopathic** [comment]
Other than the spelling fix, :+1:

**Comment by RealRancor** [comment]
Thanks for the feedback :+1: Just fixed the spelling.

**Pull request merged**